### DISCUSSION TITLE
Pooling code is released

### DISCUSSION THREAD

**Discussion by TaylorM87** [ericaltendorf/plotman#831] [Ideas]
Pooling code has released. Is there any way to get this to plot using the pooling protocol. It looks like it requires a url for the pool and a new plot create command.
chia plotnft create -u <URL> -s pool

**Comment by altendky** [comment]
Where has the pooling code been released?  Also, the command you mention is a chia command that doesn't create a plot.  plotman already has the options implemented for creating pool plots.  plotman will be released soon.

**Reply by TaylorM87** [reply]
Sorry I mean to say portable plots to a pool contract using the new portable plot creation that is currently in the main branch of chia-gui. That command creates the join to a specific pool. There are new flags for creating the portable plots like -c for the pool contract address. MadMax plotter has this built in already for their new . But I like Plotman more due to being more automated.

**Reply by altendky** [reply]
The next plotman release will support creating pool plots or original plots as well as the Chia Network plotter or the madMAx plotter.

**Reply by TaylorM87** [reply]
Oh, that's super cool. Thank you for all the work you're doing. Hopefully, I will be able to figure out how to update plotman :) I'm not great with github updating in terminal. Always leave something out.

**Reply by altendky** [reply]
You don't need to use git at all if you aren't developing plotman.  https://github.com/ericaltendorf/plotman#installation
